### DISCUSSION TITLE
column users.deleted_at does not exist when trying to send password recovery email from supabase ui

### DISCUSSION THREAD

**Discussion by jopfre** [supabase/supabase#12420] [Questions] [answered]
After running the migration scripts shown here: 

https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects

When I click Send password recovery under Authentication in the new instance's supabase ui I get a notification

Failed to send password recovery: failed to make recover request: Unable to process request

When I check the logs I see

`column users.deleted_at does not exist`

Everything works fine on the supabase instance I am migrating from so I believe the issue to be with the migration script or perhaps the new instance is now on a new version of some software?

Has anyone received this error before or have any ideas how to resolve?

**Comment by GaryAustin1** [comment]
Unless you just posted this in Discord, a user just now posted a similar issue.
It also happens to be something just added looking at the source code.
You might want to contact support.

Edit.  I was able to replicate the issue just creating a new project, creating a user in the dashboard, confirming email and then using dashboard to send password  reset.

**Reply by jopfre** [reply]
That was me who posted it in discord, I posted it here as well so its indexed.

I have resolved the issue now. See below.

**Comment by jopfre** [comment, marked as answer]
I managed to resolve the issue:

I noticed there was no column deleted_at on auth.users on my dev instance but there was on staging.

I paused and unpaused my development instance. This updated it and then I got the deleted_at column.

I then redumped the db and pushed it staging and everything works as expected.

Hope this helps someone else.

**Reply by GaryAustin1** [reply]
Supabase fixed the issue, so your instances got the new fix.
